cypress/integration/modules/curated_listing.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file
+ * Contains end-to-end tests for curated listing on ASK site.
+ */
+
+const tests = (cy) => {
+  const curatedPageUri = "/curated-listing";
+
+  describe("The sample curated listing page", () => {
+    it("successfully loads", () => {
+      cy.visit(curatedPageUri);
+    });
+
+    it("renders correctly", () => {
+      // Take a snapshot for visual diffing.
+      const percyOptions = {};
+      cy
+        .visit(curatedPageUri)
+        .get(".health-listing .row .au-display-lg")
+
+        .then(() => {
+          cy.percySnapshot("health_starter_kit_sample_curated_listing", percyOptions);
```

Comment:
@Tjocksmurf - given that this test is only performing a visual regression test, I would just add the sample Listing - Curated paragraph to our sample Standard Page content page. This will serve two purposes: 1) reduces the number of regression tests which count towards our Percy free tier limit; 2) the sample Standard Page serves as a show case for different paragraph types available in the system.